honeybee_radiance_command/_command.py

```diff
@@ -185,3 +185,95 @@ def after_run(self):
 
     def __repr__(self):
         return self.to_radiance()
+
+    @staticmethod
+    def path_checker(file_path=None, extn_list=[], file_descr=''):
+        """A utility method to check for input file path and normalize the path if present.
+        If extension list and file_descr are provided, then do additional checks to
+        ensure that the correct file type has been specified.
+
+        Args:
+            file_path: The path of the input. Will be normalized through 'typing'
+            extn_list: List of extensions, including the leading dot e.g. ['.hdr','.pic']
+                which if provided will be checked for before the path is set. If there
+                is no match, a value error will be raised.
+            file_descr: A phrase describing the file (e.g. 'Radiance HDR') that can be
+                used to compose the error message in case the extension check fails.
+        """
+
+        if file_path is None:
+            return None
+        else:
+            file_name,file_extn=os.path.splitext(file_path)
```

Comment:
missing space - you can use autopep8 of flake8 to catch these issues automatically.

```suggestion
            file_name, file_extn=os.path.splitext(file_path)
```